test/functional/apps/dashboard/dashboard_snapshots.js

```diff
@@ -59,28 +54,36 @@ export default function ({ getService, getPageObjects, updateBaselines }) {
 
       await PageObjects.dashboard.clickExitFullScreenLogoButton();
 
-      expect(percentDifference).to.be.lessThan(0.05);
+      expect(percentDifference).to.be.lessThan(0.02);
     });
 
     // FLAKY: https://github.com/elastic/kibana/issues/40173
+    // This test passes locally for LeeDr but fails every time on Jenkins
     it.skip('compare area chart snapshot', async () => {
       await PageObjects.dashboard.gotoDashboardLandingPage();
       await PageObjects.dashboard.clickNewDashboard();
       await PageObjects.dashboard.setTimepickerInLogstashDataRange();
       await dashboardAddPanel.addVisualization('Rendering Test: area with not filter');
+      await PageObjects.common.closeToast();
       await PageObjects.dashboard.saveDashboard('area');
+      await PageObjects.common.closeToast();
 
       await PageObjects.dashboard.clickFullScreenMode();
+      await screenshot.take('temp_area_chart_full_screen_1');
+      await PageObjects.common.sleep(5000);
+      await screenshot.take('temp_area_chart_full_screen_2');
       await dashboardPanelActions.openContextMenu();
+      await screenshot.take('temp_area_chart_full_screen_3');
       await dashboardPanelActions.clickExpandPanelToggle();
+      await PageObjects.common.sleep(5000);
```

Comment:
are the sleeps still needed in addition to the window resize?

Reply:
The sleeps were part of my debugging along with those additional screenshots.  See the issue about the skip line but it appears to be a Kibana bug.  But it never failed for me locally and always fails on Jenkins.  Oh, that reminds me I was going to try it on my Ubuntu 18 machine...
Since someone should eventually debug and fix whatever the problem is, I thought it was OK to leave the sleeps and additional screenshots in this PR.

Reply:
I unskipped the second test and ran it on my Ubuntu thinking it might reproduce the failure we see on Jenkins, but it didn't.

Reply:
Correction: I did just reproduce it locally by running all the dashboard tests that come before this one.  So something in a previous test is impacting this one.  I'll work on figuring that out.

Reply:
Update: I *think* I fixed the test.  The problem was that the dashboard tests generally don't use the navigate methods which put the timestamp in the URL, which means that this test was impacted by the browser cache from the previous test and the charts don't get the same colors (which, of course, makes the screenshot comparison fail).
I did leave one commented-out 1/2 second sleep in this test.  We'll see if we have any failures on opening the context menu without it.